app/assets/src/components/visualizations/heatmap/Heatmap.js

```diff
@@ -266,6 +267,28 @@ export default class Heatmap {
       `background-color: ${this.options.svgBackgroundColor}`
     );
 
+    const defs = this.svg.append("defs");
+
+    // Creates a color matrix that can recolor a black icon via a filter. See:
+    // https://semisignal.com/using-fecolormatrix-to-dynamically-recolor-icons-part-1-single-color-icons/
+    const generateColorMatrix = function(r, g, b) {
+      let rScaled = r / 255.0;
+      let gScaled = g / 255.0;
+      let bScaled = b / 255.0;
+      return `0 0 0 0 ${rScaled}
+                0 0 0 0 ${gScaled}
+                0 0 0 0 ${bScaled}
+                0 0 0 1 0`;
+    };
+
+    // Create a blue color filter to match $primary-light.
+    defs
+      .append("filter")
+      .attr("id", "blue")
+      .append("feColorMatrix")
+      .attr("type", "matrix")
+      .attr("values", generateColorMatrix(0x2b, 0x52, 0xcd));
```

Comment:
Can we make this color an option?
A string like "2b52cd" that you convert to this format here?

Reply:
I think the spacing looks good, the text is a bit close, but we tried more space yesterday and don't want the metadata to be too far from the heatmap.

Reply:
I'm guessing this is a response to the other comment below...
OK, it is your call. I thing it is a but cluttered now that is always visible (it was ok for me when it was on hover).